web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -75,9 +105,10 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
         meanIntensityGroup={highestMeanIntensityGroup}
       ></MeanIntensityGroupRollup>
       <PrepLevelCell
-        meanPrepLevel={true}
+        meanPrepLevelBoolean={true}
```

Comment:
If this is the only component using the true flag can we just make a new component for it?

Reply:
Yeah I can separate them, I was just using the structure that was already there. But I can change it into a meanPrepLevelCell component.

Reply:
Please do :)